doc/input-output-reference/src/overview/group-variable-refrigerant-flow-equipment.tex

```diff
@@ -619,7 +619,7 @@ \subsubsection{Outputs}\label{outputs-039}
 
 \paragraph{VRF Heat Pump Total Heating Rate {[}W{]}}\label{vrf-heat-pump-total-heating-rate-w}
 
-This output field is the operating total heating capacity of the variable refrigerant flow heat pump in Watts. The capacity includes any degradation due to defrost mode. This value is calculated for each HVAC system time step being simulated, and the results are averaged for the time step being reported. This value should match the sum of the individual zone terminal unit output variables for Zone VRF Air Terminal Total Heating Rate.
+This output field is the operating total heating capacity of the variable refrigerant flow heat pump in Watts. The capacity includes any degradation due to defrost mode. This value is calculated for each HVAC system time step being simulated, and the results are averaged for the time step being reported. This value should match the sum of the individual zone terminal unit heating coil output variables for Heating Coil Heating Rate.
```

Comment:
It thought this should say "sum of the individual zone terminal unit heating coil output variables for Heating Coil Heating Rate less any piping loss."

Reply:
It might just be the total of heating coil heating rate rather than total coil heating rate - piping loss.

in `CalcVRFCondenser`, `vrf.TotalHeatingCapacity` (VRF Heat Pump Total Heating Rate output variable) is computed like this
```
vrf.TotalHeatingCapacity = TotalCondHeatingCapacity * HeatingPLR * CyclingRatio;
```
And in `HeatingPLR` is like this
```
if (TotalCondHeatingCapacity > 0.0) {
    HeatingPLR = min(1.0, (TUHeatingLoad / vrf.PipingCorrectionHeating) / TotalCondHeatingCapacity);
} else {
    HeatingPLR = 0.0;
}
```

For normal cases where `TotalCondHeatingCapacity > 0.0`, 

`vrf.TotalHeatingCapacity` would just be 
```
TotalCondHeatingCapacity * ((TUHeatingLoad / vrf.PipingCorrectionHeating) / TotalCondHeatingCapacity) * CyclingRatio 
= TUHeatingLoad / vrf.PipingCorrectionHeating * CyclingRatio
```
`TUHeatingLoad / vrf.PipingCorrectionHeating` would be coil heating rate plus the piping loss.

Reply:
Right, I meant coil heating rates *plus* piping loss is the output from the outdoor unit.

Reply:
I see. Sorry I misunderstood earlier. I will modify the description.